jdbc/src/test/java/org/apache/metamodel/jdbc/HsqldbTest.java

```diff
@@ -417,4 +418,98 @@ public void testInterpretationOfNull() throws Exception {
         Connection conn = DriverManager.getConnection("jdbc:hsqldb:mem:interpretation_of_null", USERNAME, PASSWORD);
         JdbcTestTemplates.interpretationOfNulls(conn);
     }
+    
+    public void testCapitalization() throws Exception{
+      try(Connection connection = DriverManager.getConnection("jdbc:hsqldb:mem:different_types_insert", USERNAME,PASSWORD);){
```

Comment:
Coding conventions/formatting rule: There should be a space between "try" and "(".

(same rule is relevant other places in the diff, this is just an example)